DemoApp/android/build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.google.gms:google-services:3.0.0'
```

Comment:
There is `4.0.1`. Looks like this version doesn't support [new firebase versioning](https://developers.google.com/android/guides/versioning).